modules/ossm-installing-operator.adoc

```diff
@@ -28,6 +28,18 @@ For clusters without {SMProduct} instances, install the {SMProductShortName} Ope
 
 . Verify the {SMProductShortName} Operator is version 3.0, and click *Install*.
 
-. Use the default installation settings presented, and click *Install* to continue.
+. On the *Install Operator* page:
+
+.. Select *All namespaces on the cluster (default)* for the *Installation Mode*. This mode installs the Operator in the default `openshift-operators` namespace, which enables the Operator to watch and be made available to all namespaces in the cluster.
+
+.. Select *Automatic* for the *Approval Strategy*. This ensures that the future upgrades to the Operator are handled automatically by the {olm-first}. If you select the *Manual* approval strategy, {olm} creates an update request. As a cluster administrator, you must then manually approve the {olm} update request to update the Operator to the new version.
+
+.. Select an *Update Channel*.
++
+* The latest channel enables installation of the most recent stable version of the {SMProductName} 3 Operator. Currently, it is the default channel for installing the {SMProductName} 3 Operator.
```

Comment:
We are not using the `latest` channel. There is `stable` channel and `stable-<version>` for each minor version.

Reply:
@FilipB I will remove the word "latest" to avoid confusion. It can be rephrased to: "The channel enables installation of the most recent stable version of the {SMProductName} 3 Operator."

The point for "stable-<version>" is used in the second bullet point.